packages/node_modules/@node-red/nodes/core/storage/10-file.js

```diff
@@ -68,6 +68,9 @@ module.exports = function(RED) {
                     node.error(err,msg);
                     return done();
                 } else {
+                    if (value) {
+                        value = value.toString();
+                    }
```

Comment:
```suggestion
                    if (typeof value !== 'string' && value !== null && value !== undefined) {
                        value = value.toString();
                    }
```

This won't work if `value` is `0` - as that is a false-like value. Proposing changing the check to look for non-strings, but ignore null/undefined.